src/documents/InclusiveMakingAndHacking.html.md.handlebars

```diff
@@ -0,0 +1,205 @@
+
+We also consider the hacking approach to be present in one-to-one customizations, adaptations and transformations of technology that allow participation in activities or the use of technology by more diverse groups, including people with disabilities. This approach is found in groups like <a href="http://www.tetrasociety.org/" target="_blank" class="link-external">The Tetra Society of North America</a>, who create customized assistive devices for individuals.
+
+In learning settings, hacking might include classroom projects involving prototyping a product or system, or customizing devices to suit someone's unique needs.
+
+## Inclusively Designing Hackathons and Hacking
+
+It's been said about hackathons that the connections and ideas that get made between people attending are more important than any products that get produced. If hackathons are about building community, issues of inclusion, exclusion and audience diversity are paramount to their design.
+
+In thinking about inclusive design for hackathons, we must ask ourselves some of the same questions we ask in other contexts:
+
+### Asking Who Is Excluded
+
+We must consider <a href="WhatIsTheApproach.md#whoIsExcluded">who might be excluded?</a>, and how to redesign our approach to running, promoting and recruiting for our events.
+
+This is especially critical for hackathons trying to tackle social challenges faced by identifiable groups such as people with disabilities, under the principle of <a href="https://en.wikipedia.org/wiki/Nothing_About_Us_Without_Us" target="_blank" class="link-external">*Nothing About Us Without Us*</a>. Any extra work involved in broadening the reach of the event beyond the "natural" demographic will be balanced out by better outcomes and learning.
+
+### Expanding Collaboration
+
+Physical colocation is common with hackathons, but a requirement to be physically present to participate can exclude people based on ability to travel to the location or by disability.
+
+When considering the design of a hackathon-style event, consider:
+
+* Can you support remote collaboration using audio and video conferencing tools, shared document workspaces, and/or collaborative coding environments?
+* Are there alternate styles of event that would also be appropriate, such as the distributed approach of the <a href="http://globalgamejam.org/" target="_blank" class="link-external">Global Game Jam</a>?
+
+### The Curb Cut Effect and Innovation from the Margins
+
+There are a number of practical reasons to engage with individuals with "extreme" needs that falls outside the "average", along with the moral imperatives of inclusion.
```

Comment:
Somehow this sentence feels awkward to me. Is this point here that by working with diverse individuals, including people with disabilities, our designs are more innovative, resilient, flexible, and usable for all? i.e. that design for inclusion is a catalyst for innovation and it often causes a "ripple effect" of unanticipated social _and_ technical benefits